docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/2.balance-syntax.md

```diff
@@ -11,3 +11,5 @@ The `BALANCE` statements are listed as follows.
 |`BALANCE DATA STOP`|Stops the `BALANCE DATA` task.|
 |`BALANCE DATA REMOVE <host_list>`|Scales in the Nebula Graph cluster and detaches specific storage hosts.|
 |`BALANCE LEADER`|Balances the distribution of storage raft leaders in a Nebula Graph cluster.|
+|`BALANCE DATA RESET PLAN`|Reset the task. If a balance task fails to be restarted after being stopped, this command will reset all balance plan.|
```

Comment:
```suggestion
|`BALANCE DATA RESET PLAN`|Resets the task. If a balance task fails to be restarted after being stopped, this command will reset all balance plan.|
```

Reply:
Aha, I saw you helped commit the fix, thanks a lot @izhuxiaoqing !